src/encoded/static/components/body_map.js

```diff
@@ -768,31 +768,38 @@ class BodyMap extends React.Component {
-                                    role="button"
-                                    tabIndex="0"
-                                    onClick={(e) => this.chooseOrgan(e)}
-                                    onKeyPress={(e) => this.chooseOrgan(e)}
-                                    onMouseEnter={(e) => highlightOrgan(e, this.BodyList, this.CellsList, this.SystemsList)}
-                                    onMouseLeave={unHighlightOrgan}
-                                    disabled={this.state.systemFacets.indexOf(b) === -1}
-                                >
-                                    {b}
-                                </span>
-                            </li>
-                        ))}
-                        <button type="button" className="clear-organs" onClick={this.clearOrgans}>
-                            <i className="icon icon-times-circle" />
-                            Clear body map selections
-                        </button>
-                    </ul>
-                </div>
+                {this.props.displaySystems ?
+                    <div className="body-list body-list-top">
+                        <ul className="body-list-inner">
+                            {Object.keys(this.SystemsList).map((b) => (
+                                <li key={b}>
+                                    <span
+                                        id={b}
+                                        className={`body-list-element ${checkClass(this.state.selectedOrgan, b) ? 'active' : ''}`}
+                                        role="button"
+                                        tabIndex="0"
+                                        onClick={(e) => this.chooseOrgan(e)}
+                                        onKeyPress={(e) => this.chooseOrgan(e)}
+                                        onMouseEnter={(e) => highlightOrgan(e, this.BodyList, this.CellsList, this.SystemsList)}
+                                        onMouseLeave={unHighlightOrgan}
+                                        disabled={this.state.systemFacets.indexOf(b) === -1}
+                                    >
+                                        {b}
+                                    </span>
+                                </li>
+                            ))}
+                            <button type="button" className="clear-organs clear-organs-with-systems" onClick={this.clearOrgans}>
+                                <i className="icon icon-times-circle" />
+                                Clear body map selections
+                            </button>
+                        </ul>
+                    </div>
+                :
+                    <button type="button" className="clear-organs" onClick={this.clearOrgans}>
+                        <i className="icon icon-times-circle" />
+                        Clear body map selections
+                    </button>
+                }
```

Comment:
This button is so similar to the button above, it’d be nice to make them one. At first I was under the false impression that with displaysSystems true, you wouldn't see a Clear button.

Reply:
I fixed this.